src/AppKit/Enums.cs

```diff
@@ -3009,4 +3013,48 @@ public enum NSWorkspaceAuthorizationType : long  {
 		SetAttributes,
 		ReplaceFile,
 	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSTableViewStyle : long
+	{
+		Automatic,
+		FullWidth,
+		Inset,
+		SourceList
+	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSTitlebarSeparatorStyle : long
+	{
+		Automatic,
+		None,
+		Line,
+		Shadow
+	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSWindowToolbarStyle : long
+	{
+		Automatic,
+		Expanded,
+		Preference,
+		Unified,
+		UnifiedCompact
+	}
+
+	[Flags, Mac (10,16)]
+	[Native]
+	public enum NSTableViewAnimationOptions : ulong
```

Comment:
weird, it's not in the diff
it's used but not defined, maybe it's an older type ?

Reply:
`[Flags, NoiOS, Mac (0xa,0x7)]`
10.7
https://github.com/xamarin/ObjectiveSharpie/issues/139